packages/nodejs/src/ServiceClient.js

```diff
@@ -37,8 +36,8 @@ class ServiceClient extends BaseServiceClient {
   }
 
   setConcurrencyTokenAndChannelId(token, channelId) {
-    this.concurrencyManager.token = token
-    this._paymentChannelManagementStrategy.channelId = channelId
+    this.concurrencyManager.token = token;
+    this._paymentChannelManagementStrategy.channelId = channelId;
```

Comment:
here we have added semicolon , can you opine on why we need ; at some places and not other others ( its removed and its added as well )

Reply:
it was an auto linting issue and fixed in latest commit